index.html

```diff
@@ -331,18 +609,15 @@ <h2>
           <dfn>exitPointerLock()</dfn> method
         </dt>
         <dd>
-          <p>
-            Initiates an exit from pointer lock state if currently locked to a
-            target whose <a>shadow-including root</a> is this document, and
-            sends a <a>pointerlockchange</a> event when the lock state has been
-            exited.
-          </p>
-          <p>
-            The system mouse cursor must be displayed again and positioned at
-            the same location that it was when pointer lock was entered (the
-            same location that is reported in <code>screenX</code>,
-            <code>screenY</code>, when the pointer is locked).
-          </p>
+          <ol>
+            <li>
+              If <a>pointer-lock target</a> is not in the same <a>shadow-including root</a> as <a>this</a>'s document, return.
+            </li>
+            <li>
+              if currently locked to a target whose <a>shadow-including root</a> is this document <a>exit pointer lock</a>
```

Comment:
The "If" here is unnecessary, since you early-returned when it doesn't hold. So I think just `<a>Exit pointer lock</a> with the user agent's <a>pointer-lock target</a>.` suffices.

Reply:
Done.